clash-cores/src/Clash/Cores/EbTb.hs

```diff
@@ -0,0 +1,117 @@
+{-# LANGUAGE CPP #-}
+
+-- |
+--   Copyright   :  (C) 2024, QBayLogic B.V.
+--   License     :  BSD2 (see the file LICENSE)
+--   Maintainer  :  QBayLogic B.V. <devops@qbaylogic.com>
+--
+--   8b/10b encoding and decoding functions
+module Clash.Cores.EbTb where
+
+import qualified Clash.Cores.EbTb.Decoder as Dec
+import qualified Clash.Cores.EbTb.Encoder as Enc
+import Clash.Prelude
+
+-- | Data type that contains a 'BitVector 8' with the corresponding error
+--   condition of the decode function
+data DataWord
```

Comment:
The datatype could probably do with a better name, to avoid confusion between the type DataWord and it being a "data word" (vs a control word).